pom.xml

```diff
@@ -334,9 +346,22 @@
         <version>${publisher.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.android.databinding</groupId>
+        <artifactId>dataBinder</artifactId>
+        <version>1.0-rc2</version>
+      </dependency>
+
     </dependencies>
   </dependencyManagement>
 
+  <repositories>
+    <repository>
```

Comment:
We can not have a repository element in the pom if we deloy to Central. Why is this needed anyway?